spanner/spanner_snippets/spanner/spanner_create_database_with_multi_region_encryption_key.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_create_database_with_MR_CMEK]
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+func createDatabaseWithCustomerManagedMultiRegionEncryptionKey(ctx context.Context, w io.Writer, db string, kmsKeyNames []string) error {
+	// db = `projects/<project>/instances/<instance-id>/database/<database-id>`
+	// kmsKeyNames := []string{"projects/my-project/locations/locations/<location1>/keyRings/<keyRing>/cryptoKeys/<keyId>",
+	//	 "projects/my-project/locations/locations/<location2>/keyRings/<keyRing>/cryptoKeys/<keyId>",
+	//	 "projects/my-project/locations/locations/<location3>/keyRings/<keyRing>/cryptoKeys/<keyId>",
+	// }
+	matches := regexp.MustCompile("^(.+)/databases/(.+)$").FindStringSubmatch(db)
```

Comment:
issue: similar to previous--pass in instance and DB name as individual arguments. Remove the regex matching.